cypress/e2e/buyProductsAsExistingCustomer.cy.ts

```diff
@@ -0,0 +1,74 @@
+import { CHECKOUT_ELEMENTS } from "../elements/checkout/checkout-page";
+import { NAVIGATION } from "../elements/navigation";
+import { SHARED_ELEMENTS } from "../elements/shared-elements";
+import { productsToSearch } from "../fixtures/search";
+import { addItemToCart, openProductPage } from "../support/pages/product-page";
+import { waitForProgressBarToNotBeVisible } from "../support/shared-operations";
+import { payByDummyPayment } from "../support/pages/checkout-page";
+import { checkIfOrderNumberAndPaymentStatusAreCorrect } from "../support/pages/order-confirmation-page";
+
+describe("Buy product as existing user", () => {
+  let address;
+
+  before(() => {
+    cy.fixture("addresses").then(({ usAddress }) => {
+      address = usAddress;
+    });
+  });
```

Comment:
Is address used anywhere?

Reply:
Nope, it's leftovers after copy-pasting from anonymous user test